src/features/users/store/store.ts

```diff
@@ -0,0 +1,31 @@
+import { ZetkinUser } from 'utils/types/zetkin';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RemoteItem, remoteItem } from 'utils/storeUtils';
+
+export interface UsersStoreSlice {
+  userItem: RemoteItem<ZetkinUser>;
+}
+
+const initialUsersState: UsersStoreSlice = {
+  userItem: remoteItem(0),
+};
+
+const usersSlice = createSlice({
+  initialState: initialUsersState,
+  name: 'users',
+  reducers: {
+    userLoad: (state) => {
+      state.userItem.isLoading = true;
+    },
+    userLoaded: (state, action: PayloadAction<ZetkinUser>) => {
+      const user = action.payload;
+
+      state.userItem.data = user;
+      state.userItem.loaded = new Date().toISOString();
+      state.userItem.isLoading = false;
```

Comment:
By not setting the `id` of `state.userItem`, it remains `0` even after loading a user.

Reply:
I will set it here with the user.id